docs/docs/multi-core-builds.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Multi-core builds
+---
+
+Gatsby now performs the static HTML generation phase of the overall [Page HTML Generation](/docs/html-generation/) process using multi-core parallel pools of workers. This helps speed up builds by distributing build generation tasks across multiple cores on your machine.
+
+By default Gatsby creates a pool of workers equal to the number of physical cores on your machine, see [build-html.js](/docs/html-generation/#build-htmljs)
+
+In some scenarios it may be appropriate to tell Gatsby to use a different method to calculate the number of worker pools.
+
+**For example**, if you are running a Cloud server (like AWS EC2) your DevOps engineers may want to control the number of worker pools to improve the efficiency of server resource usage.
+
+### Warning
+
+You could negatively impact performance if you use this variable incorrectly. The default Gatsby setting (no env variable or `physical_cores`) is the safest option.
+
+### Setup
+
+Set the `GATSBY_CPU_COUNT` environment variable whilst running the `gatsby build` command.
+
+`GATSBY_CPU_COUNT=physical_cores` - (default) calculate the number worker pools based on the number of physical CPU cores on your machine.
```

Comment:
looks like it ought to read "number of" in each of these lines. Is that right?

Reply:
Updated :)